dartagnan/src/test/java/com/dat3m/dartagnan/spirv/SpirvChecksTest.java

```diff
@@ -86,7 +86,7 @@ public static Iterable<Object[]> data() throws IOException {
                 // {"gpu-verify/alignement/race_location.spv.dis", 1, PASS},
                 {"gpu-verify/atomics/atomic_read_race.spv.dis", 1, PASS},
                 {"gpu-verify/atomics/counter.spv.dis", 1, PASS},
-                // {"gpu-verify/atomics/definitions_atom_int.spv.dis", 1, PASS},
+                 {"gpu-verify/atomics/definitions_atom_int.spv.dis", 1, PASS},
```

Comment:
Formatting (extra space), Please also enable this case for SpirvRacesTest

Reply:
Enabled